Sources/TuistAutomation/Utilities/BuildGraphInspector.swift

```diff
@@ -127,6 +136,21 @@ public final class BuildGraphInspector: BuildGraphInspecting {
             .sorted(by: { $0.name < $1.name })
     }
 
+    public func runnableTarget(scheme: Scheme, graphTraverser: GraphTraversing) -> (Project, Target)? {
```

Comment:
Nitpick: I'd return [`ValueGraphTarget`](https://github.com/tuist/tuist/blob/main/Sources/TuistGraph/ValueGraph/ValueGraphTarget.swift#L4) instead. It contains the project, the target, and the project's path.

Reply:
👍

Reply:
Agree! Going to refactor these methods to return this instead